app/models/stats.rb

```diff
@@ -12,17 +12,23 @@ def initialize
   private
 
   def calculate_new_reg
-    from = 7.days.ago.beginning_of_day
-    to = 1.day.ago.beginning_of_day
-
-    WasteExemptionsEngine::Registration.where(submitted_at: from..to, referring_registration_id: nil).count
+    retrieve_reg_from_last_week.where(referring_registration_id: nil).count
   end
 
   def calculate_assisted_pc
-    35.4
+    assisted_reg = retrieve_reg_from_last_week.where.not(assistance_mode: nil).count
```

Comment:
[Super NIT] since we have a method for `calculate_new_reg`, why not create a `calculate_assisted_reg`?

Reply:
Thanks -- I've renamed `calculate_new_reg` to `count_new_reg` (since that's really what it does!) and added a `count_assisted_reg` method.